Sources/Highlighting/TaggedString.swift

```diff
@@ -16,6 +16,8 @@ public struct TaggedString: Hashable {
   public let untaggedRanges: [Range<String.Index>]
   
   public init(string: String, preTag: String, postTag: String, options: String.CompareOptions = []) {
+    // This string reconstruction is here to avoid a potential problems due to string encoding
```

Comment:
I would just add the example of the bug, for future readers to understand, or just mention the UT that tests it